weatherreport.cpp

```diff
@@ -46,7 +66,9 @@ namespace WeatherSpace
         {
             if (precipitation >= 20 && precipitation < 60)
                 report = "Partly Cloudy";
-            else if (sensor.WindSpeedKMPH() > 50)
+            else if (precipitation >= 60 && precipitation <= 80) // Change this condition
```

Comment:
Very high wind speed without rain will still report "light rain". Do you feel that will be in-line with what the customer wants?
Try thinking of higher level asserts, like we did in the class exercise